docs/game_sdk/Networking.md

```diff
@@ -316,16 +316,18 @@ lobbyManager.OnMemberUpdate += (lobbyId, userId) =>
   lobbyManager.UpdatePeer(peerId, newRoute);
 }
 
-// Connect to lobby with an id of 12345 and a secret of "password"
+// Connect to lobby with an id of 12345 and a secret of "password" 
+// This is used when Lobby Searching and a User wants to join remotely through this!
```

Comment:
Wording:

> "This may occur in a lobby search, when a user needs to input a password to join a lobby"

Reply:
Will Change it to that! I just wanted to give more insight in that part of the function, since that whole example imo is just snippets mashed together...?